### PR TITLE
Set up a rails scaffold

### PR DESCRIPTION
**Pull request by robin-lillqvist**
# [PT Story](https://www.pivotaltracker.com/story/show/171811161)
```
As a development team,
In order to be able to start adding features,
We would like to have a basic application structure.
```
## Changes proposed in this pull request:
* Created rails application scaffold
## What I have learned working on this feature:
* How to generate a scaffollded rails application in the current folder you are residing in.
Using: 

```
rails new . --api --database=postgresql --skip-test
```

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+
+ruby '2.5.1'
+gem 'rails', '~> 6.0.2', '>= 6.0.2.1'
+gem 'pg', '>= 0.18', '< 2.0'
+gem 'puma', '~> 4.1'
+gem 'bootsnap', '>= 1.4.2', require: false
+
+group :development, :test do
+  gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
```
Comment (blake-futchi) [inline Gemfile]: Do we really need this?

File: .ruby-version [hunk 2]
```diff
@@ -0,0 +1 @@
+ruby-2.5.1
```
Comment (blake-futchi) [inline .ruby-version:1]: why not the most updated stable ruby version?

**Review by blake-futchi — DISMISSED** [review]

File: Gemfile (hunk 1 above)
Comment (blake-futchi) [reply Gemfile]: referring to byebug

File: .ruby-version (hunk 2 above)
Comment (robin-lillqvist) [reply .ruby-version:1]: This is the latest ruby version supported by semaphore.

File: Gemfile (hunk 1 above)
Comment (robin-lillqvist) [reply Gemfile]: It's goone

**Pull request merged**